caravel/assets/javascripts/explorev2/constants.js

```diff
@@ -71,6 +74,93 @@ export const TableVizControls = (
   </div>
 );
 
+export const AreaVizControls = (
```

Comment:
i think we should create a new controls module to handle the logic around mapping control panels to viz's. i will work on a pattern for this.